x/executionlayer/client/rest/msgCreator.go

```diff
@@ -158,3 +158,130 @@ func getBalanceQuerying(w http.ResponseWriter, cliCtx context.CLIContext, r *htt
 
 	return bz, nil
 }
+
+type createValidatorReq struct {
+	ChainID                    string `json:"chain_id"`
+	ValidatorAddressOrNickName string `json:"validator_address_or_nickname"`
+	ConsPubKey                 string `json:"cons_pub_key"`
+	Moniker                    string `json:"moniker"`
+	Identity                   string `json:"identity"`
+	Website                    string `json:"website"`
+	Details                    string `json:"details"`
```

Comment:
Why don't we use [Description](https://github.com/hdac-io/friday/blob/3c822bea7fe338f612ce41ee9ede6c2e1c5cda53/x/executionlayer/types/validator.go#L75) ?

Reply:
I remember that we decided not to use double json like baseReq.

Reply:
Try not to make nested JSON struct

Reply:
But, this is not for convenience such as `baseReq`.
This is a part of abstraction indicating that they are a description of a validator.

Moreover, if we change Description struct, we have to modify all of them, and also it is hard to track the points to be modified.

And also I believe a nested JSON is easier to comprehend the meaning of the JSON than a flatten JSON.

Nevertheless, you guys still ensure the benefits are bigger, I will follow~!

Reply:
Please comment @hdac-io/developers

Reply:
https://github.com/hdac-io/friday/pull/65/commits/3ddd2ceb32b133eb8563dae0dbe5d85454120d09